### DISCUSSION TITLE
Screen/Display size. Can't find setting. Has it been obsoleted?

### DISCUSSION THREAD

**Discussion by DendrocoposMajor** [freeplane/freeplane#866] [Help]
There used to be a setting to adapt the user interface to running on different size monitors. 
I think it was located under "Tools/Preferences/Appearance".

Now I can't seem to find it. 
Has Freeplane now the ablility to do the size adaption automatically or has the setting moved elsewhere?
If moved, where as I can't seem to find it. And when did it change/move? 
Can anyone please help?

**Comment by euu2021** [comment]
See #801
